core/embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h

```diff
@@ -321,22 +411,284 @@ STATIC mp_obj_t mod_trezorcrypto_secp256k1_context_multiply(
     mp_raise_ValueError("Invalid public key");
   }
   uint8_t out[65];
```

Comment:
it's not a problem but I find it somewhat weird that we use `uint8_t` here but usually `bytes`

Reply:
You're right - changed to `uint8_t`, following `modtrezorcrypto-secp256k1.h`.